Keithley2400Sup/Keithley2400.proto

```diff
@@ -33,14 +33,14 @@ getID {
 # /// Read the voltage from the data string. Format is %f (V), %f (I), %f (R), %f (Timestamp), %f (Status)
 get_V {
    ExtraInput = Ignore;
-   out ":READ?";
+   out ":MEAS:VOLT?";
```

Comment:
Are we convinced that this is the right behaviour? According to the Keithley manual:

> :CONFigure:<function>: Configures SourceMeter for measurements on specified function. Output turns on.
> :MEASure[:<function>]?: One-shot measurement mode. Performs a :CONFigure and a :READ?.

Which implies that the output will now always be on if we are polling with `:MEAS?` commands.

@GDH-ISIS may know more about this but I am not sure that this is the behaviour we want.